### ISSUE TITLE
2.4.12 Label in Name: Editorial issues

### ISSUE THREAD

**Issue by cyns** [w3c/wcag21#835]
I agree with the idea here, but this SC is hard to parse. 
"For user interface components with labels that include text or images of text, the name contains the text presented visually."

The last clause can be read as "contains the text that is visually presented" or "contains the text and presents it visually." I think you mean the first, but I had to read it about 5 times to decide. 

The word "name" is overloaded. This use of "name" would be more clear as "accessible name" or "accessibility api name"

**Comment by awkawk** [comment]
Thank you for commenting. For more information about how the AG WG will process this issue, see the following:
- [Issue Processing](https://www.w3.org/WAI/WCAG21/comments/#issueprocessing) - How the WG will discuss the issue
- [Issue Resolution](https://www.w3.org/WAI/WCAG21/comments/#issueresolution) - Identifying and responding to the official WG response
- [Issue Notification](https://www.w3.org/WAI/WCAG21/comments/#issuenotification) - How to limit notifications from GitHub

**Comment by awkawk** [comment]
Proposed response:
Thank you for the comment. We will change the SC text editorially to read:
"For user interface components with labels that include text or images of text, the name contains the text that is presented visually."

Regarding the use of "name" the Working Group is using the term consistently sit was used in WCAG 2.0 and made a decision to not change the glossary definition. This can be reconsidered for Silver.

**Comment by awkawk** [comment]
The WG decided on the above response, so we changed the text in the comment containing the proposed response to read "[Official WG Response]". Please confirm is you are satisfied with the response within 3 days. If we haven't heard a response by then we will regard the resolution as satisfactory.
